module/zfs/dsl_bookmark.c

```diff
@@ -1651,42 +1649,40 @@ dsl_redaction_list_traverse(redaction_list_t *rl, zbookmark_phys_t *resume,
 	}
```

Comment:
line 1732: callers special-case the return value of EINTR, but I don't see where that is generated.  Did we mean to return EINTR (rather than 0) if the callback returns nonzero?